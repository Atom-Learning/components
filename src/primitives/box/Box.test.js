import { render, screen } from '@testing-library/react'
import React from 'react'

import { Box } from './'

describe(`Box component`, () => {
  it('renders', async () => {
    const { container } = render(
      <Box css={{ m: 'auto', height: 100, width: 100 }}>BOX_CHILDREN</Box>
    )
    await screen.findByText('BOX_CHILDREN')
    expect(container).toMatchSnapshot()
  })
})
