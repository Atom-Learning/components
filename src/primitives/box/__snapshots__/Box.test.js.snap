// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Box component renders 1`] = `
._Fgykb {
  margin-top: auto;
}

._bVZaHo {
  margin-right: auto;
}

._kSJetr {
  margin-bottom: auto;
}

._jbMiSP {
  margin-left: auto;
}

._SOQhS {
  height: 100px;
}

._ffzcdL {
  width: 100px;
}

<div>
  <div
    class="_ffzcdL _SOQhS _jbMiSP _kSJetr _bVZaHo _Fgykb scid-PJLV"
  >
    BOX_CHILDREN
  </div>
</div>
`;
